library/src/main/java/com/schibsted/spain/barista/internal/viewaction/AutoCompleteViewActions.java

```diff
@@ -16,7 +16,7 @@
 import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
 import static org.hamcrest.Matchers.allOf;
 
-public class AutocompleteViewActions {
+public class AutoCompleteViewActions {
 
   public static ViewAction replaceAutocomplete(@Nonnull String stringToBeSet) {
```

Comment:
It the class has been renamed to AutoComplete, the method should too, to keep the naming style.

Reply:
True! Ma' mistake